packages/techdocs-common/src/stages/generate/helpers.test.ts

```diff
@@ -147,7 +147,7 @@ describe('helpers', () => {
 
     it('should return correct target url for supported hosts', () => {
       const parsedLocationAnnotation1: ParsedLocationAnnotation = {
-        type: 'github',
+        type: 'github' as RemoteProtocol,
```

Comment:
`getRepoUrlFromLocationAnnotation` isn't very helpful in its current state as it only supports the deleted `github` and `gitlab` types. But it should be possible to use the `ScmIntegrations` to figure out which kind of integration it is instead. But that's probably a task for a dedicated PR.

Reply:
Yeah, will create a follow-up issue to track.